radis/lbl/factory.py

```diff
@@ -248,7 +248,7 @@ class SpectrumFactory(BandFactory):
         - ``"min-RMS"`` : weights optimized by analytical minimization of the RMS-error (See: [Spectral-Synthesis-Algorithm]_)
         - ``"simple"`` : weights equal to their relative position in the grid
 
-        If using the LDM optimization, broadening method is automatically set to ``'fft'``.
+        If using the LDM optimization, broadening method is automatically set to ``'fft'``. #TODO: Check if this is still true
```

Comment:
Is it?

Reply:
I checked this is deprecated, the line ``If using the LDM optimization, broadening method is automatically set to ``'fft'``. #TODO: Check if this is still true`` can be fully removed.